packages/indy-vdr/tests/indy-vdr-pool.e2e.test.ts

```diff
@@ -100,24 +98,37 @@ describe('IndyVdrPoolService', () => {
       })
 
       const response = await pool.submitWriteRequest(agentContext, request, signerKey)
+
+      expect(response).toMatchObject({
+        op: 'REPLY',
+        result: {
+          txn: {
+            protocolVersion: 2,
+            metadata: expect.any(Object),
+            data: expect.any(Object),
+            type: '1',
+          },
+          ver: '1',
+          rootHash: expect.any(String),
+          txnMetadata: expect.any(Object),
+        },
+      })
     })
   })
 
-  xdescribe('CredentialDefinition', () => {})
-
   describe('Schemas & credential Definition', () => {
     test('can write a schema using the pool', async () => {
       const pool = indyVdrPoolService.getPoolForNamespace('pool:localtest')
 
-      const dynamicVersion = `1.${Math.random() * 100}` // TODO Remove this before pushing
+      const dynamicVersion = `1.${Math.random() * 100}`
```

Comment:
@TimoGlastra this was originally added by me. I assumed we could remove this, as the CI uses a clean ledger for every run. Is my assumption correct?

Reply:
Yeah we can. But then subsequent test runs (local) will fail I think? So I thought there's not harm in keeping it in.